### PR TITLE
[BE-146] System report: osx-xcode-13.4.x

### PR DESCRIPTION
**Pull request by pgyula**

### CODE REVIEW

File: system_reports/MACOS/M1/osx-xcode-13.4.x.log [hunk 1]
```diff
@@ -7,34 +7,40 @@
...
+  11
+* 11.0 (set by /Users/vagrant/.jenv/version)
   11.0.11
   17
   17.0
   17.0.3
   openjdk64-1.8.0.292
   openjdk64-11.0.11
   openjdk64-17.0.3
+  temurin64-17.0.3
```
Comment (lpusok) [inline system_reports/MACOS/M1/osx-xcode-13.4.x.log:32]: Is both openjdk64 and temurin64 needed?

File: system_reports/MACOS/M1/osx-xcode-13.4.x.log [hunk 2]
```diff
@@ -2047,12 +2109,14 @@ zstd
 --- Available ruby versions ---
   system
 * 2.6.9 (set by /Users/vagrant/.rbenv/version)
+  2.7.2
   2.7.5
+  3.0.0
```
Comment (lpusok) [inline system_reports/MACOS/M1/osx-xcode-13.4.x.log:2114]: Were 2.7.2 and 3.0.0 requested?

Comment (pgyula) [reply system_reports/MACOS/M1/osx-xcode-13.4.x.log:2114]: yes, it was requested by customers.

File: system_reports/MACOS/M1/osx-xcode-13.4.x.log (hunk 1 above)
Comment (pgyula) [reply system_reports/MACOS/M1/osx-xcode-13.4.x.log:32]: it is for backward compatibility.

**Pull request merged**